docker/example.env

```diff
@@ -0,0 +1,5 @@
+MAIL_DOMAIN="chat.example.com"
+ACME_EMAIL="my.email@gmail.com"
+
+CERTS_ROOT_DIR_HOST="./traefik/data/letsencrypt/certs"
```

Comment:
This variable is a bit confusing if you use the non-traefik-setup... and without traefik, it is actually not used, right?